src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -640,6 +611,74 @@ export function useSwapInputsController({
     }
   );
 
+  /**
```

Comment:
NOTE: this is not all brand new logic - this is mostly moving this animatedReaction from line 812 to here so that it's ahead of another animatedReaction that reacts to maxSwappableAmount changes